templates/create-daml-app/ui/src/components/LoginScreen.tsx.template

```diff
@@ -45,16 +45,14 @@ const LoginScreen: React.FC<Props> = ({onLogin}) => {
 
   useEffect(() => {
     const url = new URL(window.location.toString());
-    const token = url.searchParams.get('token');
-    if (token === null) {
-      return;
-    }
     const party = url.searchParams.get('party');
     if (party === null) {
-      throw Error("When 'token' is passed via URL, 'party' must be passed too.");
+      return;
     }
     url.search = '';
     window.history.replaceState(window.history.state, '', url.toString());
+    const tokenCookiePair = document.cookie.split('; ').find(row => row.startsWith('DAMLHUB_LEDGER_ACCESS_TOKEN')) || '';
```

Comment:
maybe consider something like https://stackoverflow.com/a/25490531/3877993 which is probably a bit more reliable (e.g., this would match on `DAMLHUB_LEDGER_ACCESS_TOKEN2` afaict).

Reply:
Updated & tested. @da-tanabe You guys may want to update the docs to match.

Reply:
cc @alexmatson-da @sammy-da